crates/meroctl/src/cli/bootstrap/start.rs

```diff
@@ -0,0 +1,393 @@
+        let node1_server_port: u32 = 2428;
+        let node1_environment = &Environment::new(
+            RootArgs::new(
+                nodes_dir.clone(),
+                node1_name.to_owned(),
+                crate::output::Format::Json,
+            ),
+            Output::new(crate::output::Format::Json),
+        );
+
+        let node1_process = self
+            .initialize_and_start_node(
+                binary.clone(),
+                nodes_dir.to_owned(),
+                node1_log_dir.to_owned(),
+                &node1_name,
+                2528,
+                node1_server_port,
+            )
+            .await?;
+        processes.push(node1_process);
+
+        println!("Creating context in {:?}", node1_name);
+        let (context_id, public_key, application_id) =
+            StartBootstrapCommand::create_context_in_bootstrap(
+                app_path.to_owned(),
+                node1_environment,
+            )
+            .await?;
+
+        let node2_name = "node2".to_owned();
+        let node2_log_dir: Utf8PathBuf = "output/node_2_output".into();
+        let node2_server_port: u32 = 2429;
+        let node2_environment = &Environment::new(
+            RootArgs::new(
+                nodes_dir.clone(),
+                node2_name.to_owned(),
+                crate::output::Format::Json,
+            ),
+            Output::new(crate::output::Format::Json),
+        );
+
+        let node2_process = self
+            .initialize_and_start_node(
+                binary,
+                nodes_dir.to_owned(),
+                node2_log_dir.to_owned(),
+                &node2_name,
+                2529,
+                node2_server_port,
+            )
+            .await?;
+        processes.push(node2_process);
+
+        let invitee_private_key = PrivateKey::random(&mut rand::thread_rng());
+
+        self.invite_and_join_node(
+            context_id,
+            public_key,
+            invitee_private_key,
+            &node1_environment,
+            &node2_environment,
+        )
+        .await?;
+
+        println!("************************************************");
+        println!("🚀 Bootstrap finished. Nodes are ready to use! 🚀");
+
+        if demo_app {
+            println!(
+                "Application is available at https://calimero-network.github.io/core-app-template/"
+            );
+            println!("Open it in a separate window for each node.");
+            println!(
+                "Application id for node {:?} is {:?} and url is http://localhost:{}",
+                node1_environment.args.node_name,
+                application_id.to_string(),
+                node1_server_port
+            );
+            println!(
+                "Application id for node {:?} is {:?} and url is http://localhost:{}",
+                node1_environment.args.node_name,
+                application_id.to_string(),
+                node2_server_port
+            );
```

Comment:
why are we printing `application_id` for both nodes if it's the same?

the context id (and the node identity), which I'd think to be the more relevant bit, isn't also printed?

Reply:
I followed fields in the UI. Thought it may be easier for them to copy those parts like that

Reply:
They don’t need to write it anywhere and atm they will have only one context to select.

If they need it for some reason, it is printed during the bootstrap.